frontend/src/api/open-hands.types.ts

```diff
@@ -70,12 +71,21 @@ export interface AuthenticateResponse {
   error?: string;
 }
 
+export interface Repository {
```

Comment:
FYI we have an existing type `GitRepository` in `frontend/src/types/git.d.ts`!

Reply:
Thanks!

Changed the code to use the existing `GitRepository`.